src/offline/sign-message.js

```diff
@@ -0,0 +1,19 @@
+/* @flow */
+'use strict' // eslint-disable-line strict
+const common = require('../common')
+const keypairs = require('ripple-keypairs')
+const binary = require('ripple-binary-codec')
+const {validate} = common
+
+function signMessage(message: string,
+  privateKey: string
+): string {
+  validate.signMessage({message, privateKey})
+
+  const signingData = binary.encode({
+    message: message
```

Comment:
This isn't going to work: `binary.encode` expects ledger entry, or transaction json, as spoken by rippled. It silently discards fields, so you're probably turning the message into a signingData of empty bytes.

Reply:
https://github.com/ripple/ripple-binary-codec/issues/10

Reply:
You probably just want to encode the message as utf-8 ?

Reply:
With some kind of `Hash Prefix` ?

Reply:
Thank you!

Not sure how to select a `Hash Prefix`. Perhaps just encoding as utf-8 and using its hex representation is sufficient?

Reply:
@JoelKatz is the chief crypto guy @ ripple :)

Tell him your use case.

Reply:
> Perhaps just encoding as utf-8 and using its hex representation is sufficient?

Perhaps, but I don't know what you're doing, and I'm not a crypto guy. 

Ask @justmoon or @JoelKatz